mne/io/eeglab/eeglab.py

```diff
@@ -525,8 +525,13 @@ def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
     if event_id is None:
         event_id = dict()
 
-    types = [event.type for event in eeg.event]
-    latencies = [event.latency for event in eeg.event]
+    try:
+        types = [event.type for event in eeg.event]
+        latencies = [event.latency for event in eeg.event]
+    except TypeError:
```

Comment:
`len(eeg.event)` gives an error with one event unfortunatelly, I tried that before.
The types can be:
- `np.ndarray` when no events (then `.shape` is (0,) ) or if more than one event (then `.shape` is nonzero)
- `scipy.io.matlab.mio5_params.mat_struct` if only one event